composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "biodun/php-helper-methods",
+    "type": "library",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "Biodun\\PhpHelperMethods\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Biodun\\PhpHelperMethods\\Tests\\": "tests/"
+        },
+        "files": [
+            "src/Helpers/helpers.php"
+        ]
+    },
+    "authors": [
+        {
+            "name": "Biodun Bamigboye",
+            "email": "bamigboyebiodun@gmail.com"
+        } ,
+        {
+            "name": "Moffat Munene",
+            "email": "moffmu@gmail.com"
+        }
+    ],
+    "minimum-stability": "stable",
+    "require": {
+        "php": "^8.0",
+        "laravel/framework": "^9.50.2|^10.0"
```

Comment:
Let Us remove Laravel as a dependency so that It can be used in  any php application, lets avoid additional vendor files as well as much as possible.

Reply:
done